modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -254,6 +257,11 @@ export class Wallets implements IWallets {
       return this.generateMpcWallet({ multisigType: 'blsdkg', label, passphrase });
     }
 
+    // Handle distributed custody
+    if (params.isDistributedCustody && !params.enterprise) {
+      throw new Error('must provide enterprise when creating distributed custody wallet');
+    }
```

Comment:
Why to make a network call when it can be decided here not to proceed with creation if not a cold wallet?

Reply:
Yes, you should check that everything is correct before hand, otherwise if i recall correctly it will create the keys and then fail to create the wallet.

Reply:
that makes sense - updated the PR